Source/Core/Common/IniFile.h

```diff
@@ -97,6 +97,8 @@ class IniFile
    */
   bool Load(const std::string& filename, bool keep_current_data = false);
 
+  bool LoadString(const std::string configData, bool keep_current_data = false);
```

Comment:
Did you intend to pass by reference (`&` is missing if so) here?

Reply:
Nope, meant to explicilty pass here

Reply:
As in explicitly pass by value? If so, there's not much reason to do so as far as I can tell, `std::istringstream`'s constructor accepts a `const std::string&`. and `configData` is only used in the constructor, so this would copy the string when there's no need to.

Reply:
Ah, good point